float32 bla
---
float32 xCen
float32 yCen